docs/pscripts/pcreate.rst

```diff
@@ -6,8 +6,7 @@
 ``pcreate``
 -----------
 
-.. program-output:: pcreate --help
-   :prompt:
-   :shell:
+.. autoprogram:: pyramid.scripts.pcreate:PCreateCommand
```

Comment:
This either needs to be `pyramid.scripts.pcreate:PCreateCommand.parser` or if the autoprogram package isn't smart enough to parse that then after the class definition you need to pull the parser out into a module-scope variable that autoprogram can find. This means `pyramid.scripts.pcreate:_parser` and `_parser = PCreateCommand.parser`.